manage.py

```diff
@@ -99,7 +99,8 @@ def init_db(archive, force, skip_create_db=False):
 
 
 @cli.command()
-def init_mb_db():
+@click.option("--force", "-f", is_flag=True, help="Drop existing musicbrainz schema and tables.")
```

Comment:
Since this is a user-facing message, the help should say `MusicBrainz` instead of `musicbrainz.